styles.css

```diff
@@ -11,6 +11,10 @@
     color: gray;
 }
 
+#page-mod-questionnaire-questions .qcontainer .qnums ~ input[type=image] {
+    max-height: 1em;
```

Comment:
`height: 1em; width: auto;` might be better

Reply:
Have you tried injecting it on as an additional HTML if you're using the stable version instead of the main branch here?

Reply:
Adding that on additionalhtmlhead

Seems to have done the trick

Yeah, I still get a bit lost around the different versions one might encounter when using GIT, so that might not have helped

Thanks a lot for the help, I can finally do my testing without destroying my mouse scroll wheel 💪